### PR TITLE
Update profiles and fix some fallout

### PR DESCRIPTION
**Pull request by krnowak**
See https://github.com/flatcar-linux/coreos-overlay/pull/1548 for description of what happened.

Here I'm modifying some packages (dropping the use of the obsolete `elibc_{uclibc,mintlib,FreeBSD}` use flags) without moving them to coreos-overlay. The changes I have made are more or less the same changes that happened in gentoo, so there is nothing to be lost when we update the modified packages.

Needs to be merged together with https://github.com/flatcar-linux/coreos-overlay/pull/1548.

CI: http://localhost:9091/job/os/job/manifest/4566/cldsv/

### CODE REVIEW

**Comment by krnowak** [comment]
CI passed.

**Comment by krnowak** [comment]
I still need to see if some of the updates should be in updates changelog.

**Comment by krnowak** [comment]
Changelog added, rerunning CI to see if there's any fallout from the merge conflict.

File: changelog/updates/2022-01-11-profiles-update.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+- pciutils ([3.7.0](https://github.com/pciutils/pciutils/commit/864aecdea9c7db626856d8d452f6c784316a878c))
+- usbutils ([014](https://github.com/gregkh/usbutils/commit/57fb18e59cce31a50a1ca62d1e192512c905ba00))
```
Comment (tormath1) [inline changelog/updates/2022-01-11-profiles-update.md:2]: We could add an entry too for `lshw` which is a user-facing program ?

Comment (krnowak) [reply changelog/updates/2022-01-11-profiles-update.md:2]: I thought about it, but the ebuild update actually did not bring any new version of lshw. I actually updated it very recently.

Comment (tormath1) [reply changelog/updates/2022-01-11-profiles-update.md:2]: oh ok - nice then :)

**Review by tormath1 — APPROVED** [review]

**Pull request merged**